docs/source/faq/faq.md

```diff
@@ -14,7 +14,7 @@ This is a growing set of technical FAQs. The [product FAQs on the Kedro website]
 
 ## Working with Jupyter
 
-* [How can I convert functions from Jupyter Notebooks into Kedro nodes](../notebooks_and_ipython/kedro_and_notebooks.md#how-to-use-tags-to-convert-functions-from-jupyter-notebooks-into-kedro-nodes)?
+* [How can I debug a Kedro project in a Jupyter notebook](../notebooks_and_ipython/kedro_and_notebooks.md#debugging-with-debug-and-pdb)?
```

Comment:
This is fine for now, when we have all the content, I think it makes sense to have a page just talk about debugging.
It may included the content or just provide a link to:
- setup debugger in IDEs (link)
- Debugging in notebook with %load_node
- debugging in notebook with %debug %pdb

There are also debugger and breakpoints in jupyterlab (I expect works for notebook >7.0 too). %debug  or %pdb works fine for checking quickly, but I wouldn't advise to use them to debug complex logic.